eiffel-vocabulary/EiffelConfigurationOrderedEvent.md

```diff
@@ -0,0 +1,19 @@
+
+# EiffelConfigurationOrderedEvent (ArtS)
+
+The EiffeliConfigurationOrderedEvent declares that a configuration has been Ordered.
```

Comment:
Could you elaborate on what that means, exactly? How does it relate to ConfigurationApplied?